### PR TITLE
fix quest 28000 - Do the Imp-Possible

### PR DESCRIPTION
**Pull request by Snuffleclaws**

### CODE REVIEW

File: sql/updates/world/2018_04_18_01_world.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Quest 28000 - Do the Imp-possible
```
Comment (gpn39f) [inline sql/updates/world/2018_04_18_01_world.sql:1]: impsy is npc 47339.. this npc has to receive the script.. not the credit.. 47339 send on spell hit the credit spell..

File: src/server/scripts/Kalimdor/zone_felwood.cpp [hunk 2]
```diff
@@ -28,10 +28,94 @@ EndScriptData */
...
+    CreatureAI* GetAI(Creature* creature) const
+    {
+        return new npc_impsy_47365AI(creature);
+    }
+};
+
 void AddSC_felwood()
 {
+    new npc_impsy_47365();
 }
```
Comment (gpn39f) [inline src/server/scripts/Kalimdor/zone_felwood.cpp:121]: look at video: https://www.youtube.com/watch?v=fk0x_E3GdFk player has to remove all armor, weapon and cloth.. you have to check it befor cast spell

File: sql/updates/world/2018_04_18_01_world.sql (hunk 1 above)
Comment (gpn39f) [reply sql/updates/world/2018_04_18_01_world.sql:1]: I was able to attack the imp with any spell, he didn't die, but just gave me a "Right in the face! Ow ow ow..." line, then I bagged him up.

Comment (gpn39f) [reply sql/updates/world/2018_04_18_01_world.sql:1]: His health won't drop below 1 or 0.1% (regardless of what you do while on this quest),

File: src/server/scripts/Kalimdor/zone_felwood.cpp (hunk 2 above)
Comment (Snuffleclaws) [reply src/server/scripts/Kalimdor/zone_felwood.cpp:121]: Ok i will watch the video.

**Pull request closed without merging**